cs/src/core/Index/Common/Contexts.cs

```diff
@@ -148,20 +161,16 @@ public struct HybridLogRecoveryInfo
         /// Begin address
         /// </summary>
         public long beginAddress;
-        /// <summary>
-        /// Guid array
-        /// </summary>
-        public Guid[] guids;
 
         /// <summary>
-        /// Tokens per guid restored during Continue
+        /// Commit tokens per session restored during Continue
         /// </summary>
-        public ConcurrentDictionary<Guid, long> continueTokens;
+        public ConcurrentDictionary<string, CommitPoint> continueTokens;
```

Comment:
Is using a string instead of Guid a performance issue?

Reply:
Might be okay because it is not accessed in critical paths. Strings make it easier to create and reference meaningful session names. e.g. user can create sessions in code such as "s1", "s2", etc. so they dont need to persist it. If they don't provide a session name, we auto name it to a Guid converted to string. The downside is that on every commit, we have to write each session name string to disk as part of commit.